README.md

```diff
@@ -1,67 +1,126 @@
+
+## Add & remove functions (step 3):
+
+- [x] Remove all hardcoded items from the tasks array.
+- [x] Create a new JavaScript file for the new functionality.
+- [x] Implement a function for adding a new task (add a new element to the array).
+- [x] Implement a function for editing task descriptions.
+- [x] Implement a function for deleting a task (remove an element from the array).
+- [x] Implement a function for the "Clear all completed" button (use `filter()` method).
+- [x] By default new tasks should have the property `completed` set to `false` and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
+- [x] Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
+- [x] All changes to the To Do List should be saved in local storage.
+
+## Built With
+
+- Major languages: HTML, CSS, JS
+- Frameworks: none
+- Technologies used: Git, webpack
+
+## Live Demo
+
+- [Live Demo](https://hombre2014.github.io/ToDoList/dist/)
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+## Source code
 
+- [GitHub source code link](https://github.com/Hombre2014/ToDoList)
 
-To get a local copy up and running follow these simple example steps.
+## Getting Started
+
+To get a local copy up and running follow these steps.
 
 ### Prerequisites
 
+- Internet connection and browser
+- A text editor preferably ([Visual Studio Code](https://code.visualstudio.com/))
+- Browser
+
 ### Setup
 
+- For detail description of how to get started with webpack, please, look at: [webpack](https://webpack.js.org/guides/getting-started/)
+
 ### Install
 
+- [Git](https://git-scm.com/downloads)
+- [Node](https://nodejs.org/en/download/)
+
 ### Usage
 
+- Clone the repository using "`git clone git@github.com:Hombre2014/ToDoList.git`"
+- Change directory into the project folder `cd ToDoList`
+- Run `npm install`
+- Run `npm start`
+- A new browser will open automatically with application loaded
+
 ### Run tests
 
-### Deployment
+- Passed Lighthouse, webhint, Stylelint and ESLint tests
 
+### Deployment
 
+- All the files necessary for deployment are in the `/dist` folder
 
-## Authors
+## Author
 
-👤 **Author1**
+👤 **Yuriy Chamkoriyski**
 
-**Mehria-Rahmani***
-- Github: [@Merhia-Rahmani](https://github.com/Mehria-Rahmani)
-- Twitter: [@MehriaRahmani](https://twitter.com/MehriaRahmani?t=QwKwWXTVBZthv4sUrOXtfQ&s=08)
-- LinkedIn: [@mehria-rahmani](https://www.linkedin.com/in/mehria-rahmani-379211222) 
+- GitHub: [@Hombre2014](https://github.com/Hombre2014)
+- Twitter: [@Chamkoriyski](https://twitter.com/Chamkoriyski)
+- LinkedIn: [axebit](https://linkedin.com/in/axebit)
```

Comment:
- [ ] Kindly add the names of the current authors of this Project inside the ReadMe